### ISSUE TITLE
Batch cells in RandomCellSplitGridded

### ISSUE THREAD

**Issue by jeffjennings** [MPoL-dev/MPoL#166]
My implementation of train/test set splitting by randomly selecting cells in `RandomCellSplitGridded` is not well-designed for visibility datasets - it appears that randomly selecting cells in the gridded (u,v) plane is leading to artificially low cross-vallidation scores. The cells in the test sets end up being very close in the (u,v) plane to the cells in the training sets, meaning it is very easy for the model to accurately predict the test data.

The approach might work if instead of selecting individual random cells during splitting, groups of adjacent cells are selected. However, this then becomes very similar to `DartboardSplitGridded` when the batched cell sizes between the two methods are comparable (with the exception that `RandomCellSplitGridded` enforces that training sets have the highest weighted cells). In that case it's not clear that having two methods is useful.

We should compare the performance of the two methods with similar (average) cell size to find the right balance (in terms of cell size and method) between a 'noisy' estimate and a deceptively optimistic one.

**Comment by iancze** [comment]
> The cells in the test sets end up being very close in the (u,v) plane to the cells in the training sets, meaning it is very easy for the model to accurately predict the test data.

This is a great point. As I interpreted our conversation w/ @briannazawadzki and @kadri-nizam , the quality of the "prediction" to new data is relatively insensitive to regularizers, because it is able to rely so much upon nearby data in the training set.

This gets back to another discussion about "correlation scale" in the UV plane. I think we determined that the minimum correlation scale between u,v cells was dependent on the maximum size of your image, which usually corresponds to your primary beam size (which acts as a window function in the image plane). So, the correlation for a typical ALMA observation would be the spatial frequency corresponding to a 12m baseline. This would be really interesting to demonstrate/check with a mock image modified by the primary beam and then FFT'd using FourierCube.

[Credit to helpful comments from David Wilner about this].
